lake-primitives/src/types/receipts.rs

```diff
@@ -254,7 +252,76 @@ impl From<&views::ActionView> for Operation {
             views::ActionView::Delegate {
                 delegate_action,
                 signature,
-            } => todo!(),
+            } => Self::Delegate {
+                delegate_action: delegate_action.into(),
+                signature: signature.clone(),
+            },
+        }
+    }
+}
+
+#[derive(Debug, Clone)]
+pub struct DelegateAction {
+    pub sender_id: AccountId,
+    pub receiver_id: AccountId,
+    /// List of actions to be executed.
+    ///
+    /// With the meta transactions MVP defined in NEP-366, nested
+    /// DelegateActions are not allowed. A separate type is used to enforce it.
+    pub operations: Vec<NonDelegateOperation>,
```

Comment:
Why have you named it operation rather than action?

Reply:
I've named ActionReceipt as Action and I didn't like when Action included actions. So I named them Operations and here I'm following the name convention I ended up it. 

Tough I don't want to rename DelegateAction into DelegateOperation to keep the connection with nearcore 🥴 Let me know if the answer is confusing and doesn't make sense to you, please.

Reply:
I like the intent but am concerned about the divergence from `nearcore` types. Everywhere else these will be referred to as `actions`, even within this file we switch between `DelegateAction` and `(Non)DelegateOperation`. I think this could create more confusion. What do you think?

Reply:
Yeah, you are right. I'll think about it and get back to work on this in a few days.